demos/HST/download_data_HST_template.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import eureka.lib.mastDownload as md
+
+# Proposal/Program ID, can be string or int
+proposal_id = '13467'
+
+# List of one or more visit numbers
+visits = [60]
+# visits = np.arange(60,85)
+
+# Instrument name, can be upper or lower case.
+# Supported options include: WFC3, STIS, COS, or FGS.
+inst = 'WFC3'
+
+# Temporary download directory will be 'download_dir'/mastDownload/...
+download_dir = '.'
+
+# FITS file type (usually IMA, sometimes FLT)
+subgroup = 'IMA'
+
+# MAST API token for accessing data with exclusive access
+mast_token = None
+
+# Final destination of files after calling mast.consolidate
+final_dir = './HD209458/ima'
+
+# If data are public, no need to call md.login() or md.logout()
+md.login(mast_token)
+for vis in visits:
+    # Download data from MAST Archive
+    result = md.download(proposal_id, vis, inst, download_dir, subgroup)
+    if result != None:
```

Comment:
flake8 will want this to be `if result is not None:`. You should check for the other changes flake8 requests for the non-NIRISS files (once that NIRISS PR gets merged, it'll be so much easier to tell what flake8 issues were introduced by a new PR).

Reply:
Running this locally also gives me the following error traceback:
```
  File "/Users/tjbell1/Research/JWST/ERS/Eureka2/demos/HST/download_data_HST_temp.ecf", line 34, in <module>
    if result != None:
  File "/Users/tjbell1/miniconda3/envs/eureka/lib/python3.9/site-packages/astropy/table/table.py", line 3429, in __ne__
    return ~self.__eq__(other)
  File "/Users/tjbell1/miniconda3/envs/eureka/lib/python3.9/site-packages/astropy/table/table.py", line 3426, in __eq__
    return self._rows_equal(other)
  File "/Users/tjbell1/miniconda3/envs/eureka/lib/python3.9/site-packages/astropy/table/table.py", line 3474, in _rows_equal
    result = self.as_array() == other
TypeError: Cannot compare structured or void to non-void arrays.
```

Reply:
But that is addressed by changing the line to `is result is not None:`